enterprise/management/commands/seed_enterprise_devstack_data.py

```diff
@@ -57,20 +57,33 @@ class Command(BaseCommand):
     data in devstack for related Enterprise models.
     '''
 
+    def add_arguments(self, parser):
+        """
+        Entry point for subclassed commands to add custom arguments.
+        """
+        parser.add_argument(
+            '--name',
```

Comment:
Nit: `customer-name` to distinguish from usernames, etc.

Reply:
Good call. I went with "enterprise-name" because we use that elsewhere in the code.